README.md

```diff
@@ -72,3 +73,6 @@ https://github.com/aneta-s.png
 ### Resources
 
 All icons have been sourced from [Material Design Icons](https://materialdesignicons.com) and [Octicons](https://octicons.github.com/)
+
+## testing
+iam making a intial testing 
```

Comment:
Ehm.

We shouldn't commit such yolo comments.

Either add tests and create proper documentation or leave this out, please.